webpack/assets/javascripts/react_app/API.js

```diff
@@ -0,0 +1,11 @@
+import ServerActions from './actions/ServerActions';
+
+export default {
+  getStatisticsData(url) {
+    $.getJSON(url)
+      .success(
+        rawStatistics => ServerActions.receivedStatistics(rawStatistics)
+      )
+      .error(error => console.log(error));
```

Comment:
I'm guessing that hound comments on warning as well. That being said, I think the rule should be set to `error`, not `warn`, as we should not use the console for error messages (they should be displayed to the user in a way that is clearly visible, such as a notification or inside the html)